src/esm_tests/tests.py

```diff
@@ -997,7 +1069,10 @@ def format_results(info):
             if not state["comp"]:
                 compilation = "compilation failed"
 
-            run = "runs"
+            if info["actually_run"]:
+                run = "runs"
+            else:
+                run = "run files identical"
 
             if not state.get("submission_files_identical", True):
```

Comment:
For a future code refactoring, these lines are a good candidate for using `enums`:
```python
from enum import Enum

class States(Enum):
    RUNS = 0
    RUN_FILES_IDENTICAL = 1
    RUN_FILES_DIFFER = 2
    SUBMISSION_FAILED = 3
    RUN_FILES_NOT_FOUND = 4
    RUN_FAILED = 5
``` 
You can then use these constants instead of a dictionary. It will also make it more compatible across other functions. But at this point it is not that important.

Reply:
Can you clarify how would you use `enum` to improve this part of the code? I am not able to understand it just with the info you provided. I understand the class you built, but how would you use it and how is it superior to the dictionary?